src/client/components/LeaderboardList/LeaderboardList.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { withKnobs, text, number, object } from '@storybook/addon-knobs';
+import ima from '../../assets/images/avatar1-ce.png';
+import LeaderboardItem from './LeaderboardItem';
+
+export default {
+  title: 'Test Leaderborad Component',
+  decorators: [withKnobs],
+};
+
+export const TestLeaderboradComp = () => {
+  return (
+    <ul>
+      <LeaderboardItem
+        id={number('id', 1)}
+        avatar={text('src', ima)}
```

Comment:
Just curious, what `ima ` stands for? It's also in the import but I can't find the element -I understand is referred to the images-.

Reply:
it's my mistake i was testing the image URL, it should be named as image, and it's image that i imported from the folder image

Reply:
ima is stands form image he import it from  images folder, but he just name it as ima... but I think its better if nameit image instead of  just ima